src/main/java/io/codelex/parbaudesDarbs/exercise3/Joiner.java

```diff
@@ -0,0 +1,18 @@
+package io.codelex.parbaudesDarbs.exercise3;
+
+public class Joiner {
+    private String seperator;
+
+    public Joiner(String seperator) {
+        this.seperator = seperator;
+    }
+
+    public <T> String join(T... items) {
+
+        StringBuilder result = new StringBuilder();
```

Comment:
Šis ir ok, bet varētu uzlabot -  ir tāda metode:
```
List<String> list = Arrays.asList("a","b","c");
        String result = String.join(",", list);
```
Pamēģini uzrakstīt kodu, lai tas izmanto String.join metodi